CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ All notable changes to this project will be documented in this file. Changes not
 ## Changed
 
 - Set the version of the HTTP Server based in the project version in the **Info.plist** for macOS, iOS and tvOS platforms. ([#416](https://github.com/httpswift/swifter/pull/416)) by [@Vkt0r](https://github.com/Vkt0r)
+- Update `HttpParser` so it percent-encodes the URL components before initializing `URLComponents`
```

Comment:
Can you please follow the template guideline for the entry in the CHANGELOG. I think you're missing something like this (([#416](https://github.com/httpswift/swifter/pull/416)) by [@Vkt0r](https://github.com/Vkt0r)) after the end dot.  See the rest of the entries in the CHANGELOG for references.